content/07.definitions.md

```diff
@@ -0,0 +1,12 @@
+## Definitions {.page_break_before}
+
+### Unsupervised learning: 
+Machine learning algorithms which can learn features from unlabeled training data (e.g. datasets where the samples do not have disease or phenotype labels) to predict the class or phenotype of new or unseen test data are part of unsupervised learning. Examples of unsupervised learning include principal component analyses, multidimensional scaling, UMAP, tSNE, k-means clustering etc [TODO - add REFs].
```

Comment:
```suggestion
Machine learning algorithms which can learn features from unlabeled training data (e.g. datasets where the samples do not have disease or phenotype labels) to predict the class or phenotype of new or unseen test data are part of unsupervised learning. Examples of unsupervised learning include principal component analyses, multidimensional scaling, UMAP, t-SNE, k-means clustering etc [TODO - add REFs].
```

Reply:
I looked at the dictionary file and i think I added "t" and "SNE" so that "t-SNE" passes the spellcheck but "tSNE" does not. Let's go with "t-SNE" as the spelling since that's the "official" acronym.